### ISSUE TITLE
[Bug]: resources not deleting after deleting project

### ISSUE THREAD

**Issue by wp-coin** [coollabsio/coolify#1619]
### Description

after deleting a project, the resources are not deleted, but there is no indication in the settings what services are ruining

### Minimal Reproduction (if possible, example repository)

after deleting projects from the coolify projects and also using the command line 

``` 
docker exec -ti coolify bash -c "php artisan services:delete"
```

to delete servers and services , resources like phpmyadmin and mariadb are still active on the server and accessible from domain name

### Version

beta.183

**Comment by techwithanirudh** [comment]
I'm having the same issue

**Comment by andrasbacsai** [comment]
`services:delete` command will be fixed to delete the resources properly - there was a small change in the process and forgot to update there.

Also, if I understand right, you deleted the resource from the UI and it was not deleted properly?
Then you tried to delete with the `services:delete` command, right?

**Comment by fipnooone** [comment]
@andrasbacsai looks like that some resources are not removed when deleting a project. I've deleted projects (Gitea and Metube from templates), but `docker ps -a` shows `ghcr.io/alexta69/metube:latest`, `gitea/gitea:latest`, `mysql:8.0` with status `Up <time> (healthy)`. I've managed to remove them via `docker exec -ti coolify bash -c "php artisan services:delete"`, but i think resources should be deleted from the UI, right?
